### ISSUE TITLE
Update for 2025 season

### ISSUE THREAD

**Issue by witnessmenow** [witnessmenow/F1-Arduino-Notifications#13]

**Comment by cnoorman** [comment]
Hi Brian, would be nice if you could update for new season!
Cheers

**Comment by Bojouoss** [comment]
Hello, i've made the update for my own use and created a pull request.
I'm not used to how github works so feel free to correct the pull request :)

In the commit, i've also modified the code to display the circuit image until 3 days before the race so it can be displayed more often.
There are a lot of back-to-back races now so at least the image is used for a few days.

**Comment by LykleSchepers** [comment]
Is there an update for the 2026 season?

**Comment by Bojouoss** [comment]
There are 2 pull requests pending that can do this.
One is the update for 2026, the other is an update to fetch dynamically the calendar for the current year so if no new circuit is added or renamed you won't have to reflash your device.

**Comment by LykleSchepers** [comment]
Yes, I cloned the update for 2026, tried to compile it with Studio, failed. Errors beyond my capability. I am not a real coder, just a guy trying to get it to work for me.

**Comment by Bojouoss** [comment]
you can clone this one and give it a try https://github.com/Bojouoss/F1-Arduino-Notifications/tree/Dynamic_update

I'm using it right now :) 

<img width="1536" height="2048" alt="Image" src="https://github.com/user-attachments/assets/1e1f1ff8-4c9d-4e9c-b475-f401b51f6bed" />

**Comment by LykleSchepers** [comment]
I must have something wrong in my Studio config or such. I keep getting errors when compiling. 

Like this:
F1-Notifications/cheapYellowLCD.h:141:84: error: invalid conversion from 'void (*)(PNGDRAW*)' {aka 'void (*)(png_draw_tag*)'} to 'int (*)(PNGDRAW*)' {aka 'int (*)(png_draw_tag*)'} [-fpermissive]
     int rc = png.open((const char *)imageFileUri, myOpen, myClose, myRead, mySeek, PNGDraw);
                                                                                                                                         ^~~~~~~

No idea if this is relevant, but my hw version is a bit weird. It has 2 USB ports, but I have to use CYD to flash, because with CYD2USB it does not work. This si when flashing the standard version via the webpage.

**Comment by digbet** [comment]
I'm getting the same as Lykle.

**Comment by witnessmenow** [comment]
Should be addressed in #16, thanks to @Bojouoss  (Awesome display!)

have created #19 for @LykleSchepers issue
